mxtests.ini

```diff
@@ -0,0 +1,125 @@
+# If mxcheckouts.txt says we want to use plone.volto, then the generated
+# requirements-mxdev.txt will have this line:
+# plone.volto [test] -> mxdev disabled (source)
+# This would mean the 'test' extra of plone.volto will not get installed.
+# So we need to define the extras that we want in case of a checkout.
+
+[diazo]
+extras = test
+
+[plone.api]
+extras = test
+
+[plone.app.caching]
+extras = test
+
+[plone.app.contentlisting]
+extras = test
+
+[plone.app.contenttypes]
+extras = test
+
+[plone.app.discussion]
+extras = test
+
+[plone.app.event]
+extras = test
+
+[plone.app.iterate]
+extras = test
+
+[plone.app.layout]
+extras = test
+
+[plone.app.linkintegrity]
+extras = test
+
+[plone.app.portlets]
+extras = test
+
+[#    plone.app.robotframework]
```

Comment:
typo? 🤔

Reply:
Ah, well spotted.  I did this with a script or some bash line (it was a while ago) with `tests.cfg` as input.  This line in `tests.cfg` thwarted that script:

```
#    plone.app.robotframework [test]
```

Fixed by properly commenting it out in `mxtests.ini` as well.